app/controllers/todo_items_controller.rb

```diff
@@ -2,7 +2,7 @@ class TodoItemsController < ApplicationController
   before_action :get_resources
   
   def index
-    @todo_items = policy_scope(TodoItem)
+    @todo_items = policy_scope(TodoItem).where(todo_list: @todo_list)
```

Comment:
Wygodniej by było chyba zawołać to przez Punditowy scope (tak jak pozywałem dzisiaj na szkoleniu):
```ruby
@todo_items = policy_scope(@todo_list.todo_items)
```